src/screens/CreatePost/index.js

```diff
@@ -18,10 +19,9 @@ import {
   Text,
   View
 } from 'react-native';
-import {Image} from 'react-native-compressor';
-import {launchCamera, launchImageLibrary} from 'react-native-image-picker';
 import {openSettings} from 'react-native-permissions';
 import {showMessage} from 'react-native-flash-message';
+import {launchCamera, launchImageLibrary} from 'react-native-image-picker';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The order of imports has been changed and `openSettings` from 'react-native-permissions' has been moved up. This doesn't affect the functionality but it's good to keep the imports organized, usually by their source or purpose.
```diff
- import {Image} from 'react-native-compressor';
- import {launchCamera, launchImageLibrary} from 'react-native-image-picker';
- import {openSettings} from 'react-native-permissions';
+ import {openSettings} from 'react-native-permissions';
+ import {showMessage} from 'react-native-flash-message';
+ import {launchCamera, launchImageLibrary} from 'react-native-image-picker';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->